utilities/implementation-kit/data-migration/Tenant _CreationOrUpdation/folder_for_tenant.py

```
@@ -0,0 +1,470 @@
+    filename1 = "data/"+state_code+"/"+name.lower().replace(" ", "")+"/ws-services-calculation/WCBillingSlab.json"
+    os.makedirs(os.path.dirname(filename1), exist_ok=True)
+    
+    with open(filename1, 'w', encoding='utf-8') as f:
+        json.dump(rdata, f, ensure_ascii=False, indent=4)
+
+
+def create_json_penalty(state_code,name,penalty):
+    penaltyset=0
+    try:
+        penaltyset=int(penalty)
+    except:
```

Ruff

Do not use bare `except`

(E722)